undertow/src/main/resources/schema/wildfly-undertow_10_0.xsd

```diff
@@ -316,6 +316,7 @@
         </xs:attribute>
         <xs:attribute name="disable-console-redirect" use="optional" type="xs:boolean" default="false"/>
         <xs:attribute name="queue-requests-on-start" type="xs:boolean" default="true"/>
+        <xs:attribute name="active-request-tracking-enabled" type="xs:boolean" default="false"/>
```

Comment:
We don't generally modify legacy (i.e. already released) schemas, unless there was a bug in the originally release version.

Reply:
Right. Sorry. That was noise left from an experiment. I'll clean it up.

Reply:
@pferraro Do we need wildfly-undertow_14_0.xsd?

Reply:
13.0 is new as of WF27, so we don't need a new schema.  The same commit also incremented the model version, so there is no need to bump that either.  Interesting, this commit: https://github.com/wildfly/wildfly/commit/a1f9ad78f6c960e824a2ca007a40593706c2d060
... appears to have incremented the model version from 11.0.0 -> 13.0.0, so it seems there was never a 12.0.0 model.  That commit should never have been merged.
See: https://issues.redhat.com/browse/WFLY-15850

Reply:
FYI, I've filed https://issues.redhat.com/browse/WFLY-15850 to fix this before WF27 is released.
I don't mean that you need to address this here, but just be aware that whatever transformers you are introducing as part of this commit will need to be adjusted accordingly once WFLY-15850 is fixed.